keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandler.kt

```diff
@@ -58,39 +58,38 @@ class SecurityGroupHandler(
   override fun current(resource: Resource<SecurityGroup>): SecurityGroup? =
     cloudDriverService.getSecurityGroup(resource.spec)
 
-  override fun upsert(resource: Resource<SecurityGroup>, diff: DiffNode?) {
+  override fun create(resource: Resource<SecurityGroup>) {
```

Comment:
Instead of implementing `upsert` I now implement separate `create` and `update` on the handler.